examples/rename_brainvision_files.py

```diff
@@ -49,45 +66,77 @@
 examples_dir = op.join(data_path, 'Brainvision')
 
 ###############################################################################
-# Step 2: Rename the recording
-# ----------------------------------
+# Rename the recording
+# --------------------
 # Above, at the top of the example, we imported
 # :func:`mne_bids.copyfiles.copyfile_brainvision` from
-# the MNE-BIDS ``utils.py`` module. This function takes two arguments as
-# input: First, the path to the existing .vhdr file. And second, the path to
-# the future .vhdr file.
+# the MNE-BIDS ``mne_bids.copyfiles.py`` module. This function takes two
+# main inputs:
+# First, the path to the existing ``.vhdr`` file. And second, the path to
+# the future ``.vhdr`` file.
+#
+# With the optional ``verbose`` parameter you can furthermore determine how
+# much information you want to get during the procedure.
 #
 # :func:`mne_bids.copyfiles.copyfile_brainvision` will then create three new
-# files (.vhdr, .vmrk, and .eeg) with the new names as provided with the second
-# argument.
+# files (``.vhdr``, ``.vmrk``, and ``.eeg``) with the new names as provided
+# with the second argument.
 #
 # Here, we rename a test file name:
 
-vhdr_file = op.join(examples_dir, 'test_NO.vhdr')
+# Rename the file
+vhdr_file = op.join(examples_dir, 'Analyzer_nV_Export.vhdr')
 vhdr_file_renamed = op.join(examples_dir, 'test_renamed.vhdr')
-copyfile_brainvision(vhdr_file, vhdr_file_renamed)
+copyfile_brainvision(vhdr_file, vhdr_file_renamed, verbose=True)
 
-###############################################################################
-# Step 3: Assert that the renamed data can be read by a software
-# --------------------------------------------------------------
-# Finally, let's use MNE-Python to read in both, the original BrainVision data
-# as well as the renamed data. They should be the same.
+# Check that MNE-Python can read in both, the original as well as the renamed
+# data. They should be the same.
 raw = mne.io.read_raw_brainvision(vhdr_file)
 raw_renamed = mne.io.read_raw_brainvision(vhdr_file_renamed)
 
+# If we can run the following without error, everything is fine
 assert_array_equal(raw.get_data(), raw_renamed.get_data())
```

Comment:
for me you are mixing here a unit test and a tutorial. User will assume that you do the right thing so this seems superfluous.